torch/_dynamo/optimizations/distributed.py

```diff
@@ -88,17 +96,22 @@ def compile_fn(self, gm: fx.GraphModule, example_inputs: List[torch.Tensor]):
             if node.op == "call_module":
                 target = gm.get_submodule(node.target)
                 for name, p in target.named_parameters():
-                    if p.requires_grad:
+                    param = target.get_parameter(name)
+                    if p.requires_grad and not self._ignore_parameter(param):
                         buckets[0].size += p.storage().nbytes()
-                        # TODO correct FQ name?
-                        buckets[0].params.append(f"{node}_{name}")
+                        buckets[0].params.append(f"{node.target}_{name}")
```

Comment:
do we still need this?

Reply:
it is just useful for visualization purposes.  see the debug output on the next PR in this stack.  the buckets table is printed using this string.